frontend/src/components/dashboard/aliases/AliasList.module.scss

```
@@ -4,11 +4,11 @@
 
 .controls {
   display: grid;
-  grid-template-columns: 1fr auto auto;
+  grid-template-columns: auto 1fr auto;
   row-gap: $spacing-md;
   grid-template-areas:
-    "categoryFilter newAliasButton"
-    "stringFilter stringFilter";
+    "stringFilterToggle categoryFilter  newAliasButton"
```

Heh, channelling my inner Luke :)

```suggestion
    "stringFilterToggle categoryFilter newAliasButton"
```

---

Revised in eb0a0e26056ddc189603b149603d1368e575f4d9